src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -2157,7 +2182,25 @@ public static Map<String,String> getInvertiblePinMap(Series series, Unisim unisi
 	 * instances from which to copy the implementation. If targeting the top instance, use an 
 	 * empty String ("") as the destination instance name.
 	 */
-	public static void copyImplementation(Design src, Design dest, boolean lockPlacement, 
+	public static void copyImplementation(Design src, Design dest, boolean lockPlacement,
+										  boolean lockRouting, Map<String,String> srcToDestInstNames) {
+		copyImplementation(src, dest, false, false, lockPlacement, lockRouting, srcToDestInstNames);
+	}
+
+	/**
+	 * Copies the logic and implementation of a set of cells from one design to another with additional flags to control copying nets.
+	 * @param src The source design (with partial or full implementation)
+	 * @param dest The destination design (with matching cell instance interfaces)
+	 * @param copyStaticNet Flag indicating if static nets should be copied
```

Comment:
It only copies the portion of static nets inside the designated part, correct?

Reply:
yes, it traces only from the static sink within the target cell.